src/main/java/ca/sait/vezorla/service/PaypalServices.java

```diff
@@ -0,0 +1,71 @@
+package ca.sait.vezorla.service;
+
+import com.paypal.api.payments.*;
+import com.paypal.base.rest.APIContext;
+import com.paypal.base.rest.PayPalRESTException;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Services to handle Paypal
+ * payments.
+ *
+ * Taken from: https://github.com/Java-Techie-jt/spring-boot-paypal-example/blob/master/src/main/java/com/javatechie/spring/paypal/api/PaypalService.java
+ */
+@Service
+public class PaypalServices {
+
+    private APIContext apiContext;
+
+    public PaypalServices(APIContext apiContext){
+        this.apiContext = apiContext;
```

Comment:
Can this be `@AllArgsConstructor`? I tried this with `CustomerClientUtil` and it didn't work

Reply:
Done